### PR TITLE
Add inter-branch merges for arcade's repos

### PR DESCRIPTION
**Pull request by mmitche**
- arcade: release/3.x -> master
- arcade-services: production -> master
- arcade-pool-provider: production -> master
- helix-service: production -> master

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1850,6 +1850,25 @@
         }
       }
     },
+    // Automate opening PRs to merge services' production branches into master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/arcade-services/blob/production/**/*",
+        "https://github.com/dotnet/arcade-pool-provider/blob/production/**/*",
+        "https://github.com/dotnet/helix-service/blob/production/**/*",
```
Comment (riarenas) [inline Maestro/subscriptions.json]: I don't recognize this one. Is it supposed to be https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service?

Comment (mmitche) [reply Maestro/subscriptions.json]: This is the inter-branch merges, not mirroring.

Comment (riarenas) [reply Maestro/subscriptions.json]: Yeah, just wondering then, since that github repo doesn't exist

Comment (riarenas) [reply Maestro/subscriptions.json]: Synced up. The repo exists, I just don't have permissions to see it. 😶

**Pull request merged**